src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -535,4 +539,28 @@ private function isAbsolutePath(string $file): bool
     {
         return strspn($file, '/\\', 0, 1) || (\strlen($file) > 3 && ctype_alpha($file[0]) && ':' === $file[1] && strspn($file, '/\\', 2, 1)) || null !== parse_url($file, PHP_URL_SCHEME);
     }
+
+    /**
+     * @return FilesystemLoader[]
+     */
+    private function getFilesystemLoaders(): array
+    {
+        if (null !== $this->filesystemLoaders) {
+            return $this->filesystemLoaders;
+        }
+        $this->filesystemLoaders = [];
+
+        $loader = $this->twig->getLoader();
+        if ($loader instanceof FilesystemLoader) {
+            $this->filesystemLoaders[] = $loader;
+        } elseif ($loader instanceof ChainLoader) {
+            foreach ($loader->getLoaders() as $l) {
+                if ($l instanceof FilesystemLoader) {
```

Comment:
what if it's another chain loader? Is that possible?

Reply:
yep, it is possible for someone tagging its own chain loader, but that would be wierd since `TwigLoaderPass` already do that job. Do you have some example where nesting chain loaders is useful?

Reply:
nope :) just curious if we should deal with it. Following the code it could happen :man_shrugging: